freeswitch_click2dial/README.rst

```diff
@@ -0,0 +1,138 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+FreeSWITCH Click2Dial
+=====================
+
+The technical name of this module is *freeswitch_click2dial*, but this module
+implements much more than a simple *click2dial* ! This module adds 3
+functionalities:
+
+1) It adds a *Dial* button in the partner form view so that users can directly
```

Comment:
you should probably split:

* summary as introduction here
* detailed operations in usage (with enumerated list and if possible screenshots)
(nice to have)